src/main/java/ch/wisv/areafiftylan/exception/AreaFiftyLANExceptionLogLevelEnum.java

```diff
@@ -0,0 +1,9 @@
+package ch.wisv.areafiftylan.exception;
+
+enum AreaFiftyLANExceptionLogLevelEnum {
+    ERROR,
```

Comment:
E.g. add a constructor here which is a consumer of a string. Then you pass the log reference on the creation of the enum value